scripts/scil_flip_surface.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to flip and reverse cortical surface (vtk or freesurfer).
+
+Best usage for FreeSurfer to LPS vtk (for MI-Brain):
+!!! important FreeSurfer surfaces must be in their respective folder !!!
+> mris_convert --to-scanner lh.white lh.white.vtk
+> scil_flip_surface.py lh.white.vtk lh_white_lps.vtk -x -y
+"""
+
+import argparse
+
+from trimeshpy.io import load_mesh_from_file
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('surface',
+                   help='Input surface (FreeSurfer or supported by VTK).')
+
+    p.add_argument('out_surface',
+                   help='Output flipped surface (formats supported by VTK).')
+
+    p.add_argument('-x', action="store_const", const=-1, default=1,
```

Comment:
Our 3 flipping scripts now use the convention.
```
    p.add_argument('axes',
                   choices=['x', 'y', 'z'], nargs='+',
                   help='The axes you want to flip. eg: to flip the x '
                        'and y axes use: x y.')
```

I think you should use that for consistency. I think your line `flip = [args.x, args.y, args.z]` would be fairly easy to adapt

Reply:
Agree